### ISSUE TITLE
I can not get SN Code

### ISSUE THREAD

**Issue by Singosgu** [darryncampbell/darryncampbell-cordova-plugin-intent#138]
Hi Darryncampbell:

I'm a communitor of GreaterWMS--An open source warehouse management system .
GreaterWMS Link: [Github](https://github.com/Singosgu/GreaterWMS)

Now, I wanna to get the PDA SN code , but failed .
How I can get SN code by this plugin?

**Comment by Singosgu** [comment]
Currently, We use this plugin in our WMS , Our user can use zebra scanner to scan the barcode . 
cordova template link: [GreaterWMS Cordova](https://github.com/Singosgu/GreaterWMS/tree/master/templates/src-cordova)

**Comment by darryncampbell** [comment]
Hi, The Android serial number API was changed by Google in A10 to prevent unprivileged apps from accessing the serial number.  Zebra devices do expose the SN through a different API but that is based on a content provider so this plugin could not access it.  More info and native code samples for the Zebra SN API are here: https://developer.zebra.com/blog/device-identifiers-zebra-devices-running-android-10-%25E2%2580%2593-complete-guide

**Comment by Singosgu** [comment]
Hi , Could I build it in a cordova plugin?

**Comment by darryncampbell** [comment]
> Hi , Could I build it in a cordova plugin?

Yes, this plugin is released under MIT so there are no restrictions
